strada/parser.js

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env node
+
+'use strict';
+const Parser = require('../.lib/parser.js');
+
+/**
+ * Recognizes the accesses to the platform Strada Lex
+ * @param  {Object} parsedUrl an object representing the URL to analyze
+ *                            main attributes: pathname, query, hostname
+ * @param  {Object} ec        an object representing the EC whose URL is being analyzed
+ * @return {Object} the result
+ */
+module.exports = new Parser(function analyseEC(parsedUrl, ec) {
+  let result = {};
+  let pathname = parsedUrl.pathname;
+  let path = parsedUrl.path;
+  // uncomment this line if you need parameters
+  //let param = parsedUrl.query || {};
+
+  // use console.error for debuging
+  console.error(parsedUrl);
+
+  let match;
+
+  if ((match = /^\/DBPRO\/FR\/([a-z]+)\/html\/([a-z]+)\/([a-z\_]+)\/([0-9a-z\-\/\_\.\~]+)$/i.exec(pathname)) !== null) {
+    //DBPro/fr/AdvancedSearch/html/getForm/se_rech/20170407-prod-2392-58e7668d6e9ee4-75323819
+    //DBPro/FR/Search/html/breadcrumb/se_src_publ_jur/jur_int_oit/1/20170407-prod-6142-58e762ea352ad7-37590738
+    //DBPro/FR/Document/html/getDocFromDbpro/se_src_publ_jur/epo_D0016_16/0/20170407-prod-8670-58e7709a503e47-24424947
+    //DBPro/FR/Document/html/getDocFromDbpro/se_src_publ_leg/ojeu_2017.080.01.0001.01/0/20170327-prod-6774-58d919d2d7cb68-36251222
+    //DBPro/FR/Document/html/getDocFromDbpro/se_legi_chrono/DirParleuretCons-20161214-Art1~1/0/20170323-prod-4179-58d3d73d46a4a9-81798019
+    //DBPro/FR/Search/html/breadcrumb/se_legi_chrono/CHRONO_2135547/1/20170323-prod-5694
+    //DBPro/FR/Search/html/breadcrumb/se_src_publ_leg/leg_eur_jo_3_20170325_80/1/20170327-prod-7054-58d90a3b70c6c2-00242421
+    //DBPro/FR/Document/html/getDocFromDbpro/se_legi/ACT_LV_678066-19570325-Art1/0/20170323-prod-5871-58d3ce05ce7744-49426366
+    //DBPro/FR/Document/html/getDocFromDbpro/se_mono/10YEREG_003/0/20170310-prod-2861-58c288eaac2c57-38109236
+    result.mime = 'HTML';
+
+    switch (match[1]) {
+    case 'AdvancedSearch':
+      result.unitid   = match[1];
```

Comment:
"AdvancedSearch" is not a resource identifier, I guess this is an error in the analysis. In this case we don't have any identifier.